src/Sentry/Internal/Enricher.cs

```diff
@@ -86,6 +86,19 @@ public void Apply(IEventLike eventLike)
         eventLike.Contexts.App.StartTime ??= ProcessInfo.Instance?.StartupTime;
         eventLike.Contexts.Device.BootTime ??= ProcessInfo.Instance?.BootTime;
 
+        try
+        {
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                eventLike.Contexts.App.InForeground = ProcessInfo.Instance?.ApplicationIsActivated(_options);
+            }
+        }
+        catch
+        {
+            //If RuntimeInformation.IsOSPlatform(OSPlatform.Windows) throws we are not on Windows.
```

Comment:
but there's a guard, so why would it throw?

Please catch the exception and log, lets not make assumptions and hide it,makes it hard to debug later

Reply:
On lime 42 in the same file, it says that: _RuntimeInformation.* throws on .NET Framework on macOS/Linux_